api/src/main/java/org/pmiops/workbench/exceptions/ExceptionAdvice.java

```diff
@@ -0,0 +1,33 @@
+package org.pmiops.workbench.exceptions;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ControllerAdvice
+public class ExceptionAdvice {
+
+  private static final Logger log = Logger.getLogger(ExceptionAdvice.class.getName());
+
+  private static final String DEFAULT_ERROR_VIEW = "error";
+
+  @ExceptionHandler({Exception.class})
+  public String serverError(Exception e) {
+    if (e.getClass().getPackage().getName().equals(
```

Comment:
I've not used the ControllerAdvice annotation before, but is it possible that this code swallows exceptions that don't match ExceptionAdvice package? For example: what will happen if a RuntimeException is thrown?

Reply:
RuntimeException extends Exception (and our exceptions extend RuntimeException). So we're coverered there.

If some other Throwable (e.g. OutOfMemoryError) is thrown, I suspect that AppEngine will log something, and we are better off not trying to catch errors like those.